src/app/shared/services/layout.service.ts

```diff
@@ -1,42 +1,49 @@
 import { Injectable } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
-import { distinctUntilChanged } from 'rxjs/operators';
+import { BehaviorSubject, distinctUntilChanged } from 'rxjs';
 
 export interface FullFrameContent {
-  expanded: boolean,
+  active: boolean,
   canToggle: boolean,
+  animated: boolean,
 }
 
 @Injectable({
   providedIn: 'root'
 })
 export class LayoutService {
   // Service allowing modifications of the layout
-  private fullFrameContent = new BehaviorSubject<FullFrameContent>({ expanded: false, canToggle: true });
+
+  private initialized = false;
+
   /** Expands the content by hiding the left menu and select headers */
-  fullFrameContent$ = this.fullFrameContent.pipe(distinctUntilChanged((a, b) => a.expanded === b.expanded && a.canToggle === b.canToggle));
+  private fullFrame = new BehaviorSubject<FullFrameContent>({ active: true, canToggle: false, animated: false });
+  fullFrame$ = this.fullFrame.pipe(distinctUntilChanged((a, b) => a.active === b.active && a.canToggle === b.canToggle));
 
-  private showTopRightControls = new BehaviorSubject(true);
-  readonly showTopRightControls$ = this.showTopRightControls.pipe(distinctUntilChanged());
+  private showTopRightControls = new BehaviorSubject(false);
+  showTopRightControls$ = this.showTopRightControls.pipe(distinctUntilChanged());
```

Comment:
Usage of observables is in the end required because there's no other way to detect service prop changes in components :shrug:

Reply:
Of course, `layoutInitialized = this.layoutService.initialized` would not be updated... right using observable is the only way here.